requirements.txt

```diff
@@ -0,0 +1,77 @@
+amqp==2.4.2
+asn1crypto==0.24.0
+attrs==19.1.0
+backports.functools-lru-cache==1.5
+backports.ssl-match-hostname==3.7.0.1
+bcrypt==3.1.6
+billiard==3.5.0.2
+boto3==1.9.107
+botocore==1.12.107
+celery==4.1.1
+certifi==2018.11.29
+cffi==1.12.2
+chardet==3.0.4
+cheroot==6.5.4
+CherryPy==11.0.0
+Click==7.0
+click-plugins==1.0.4
+git+https://github.com/girder/covalic.git@girder3
+cryptography==2.6.1
+decorator==4.3.2
+diskcache==3.1.1
+docker==3.7.0
+docker-pycreds==0.4.0
+docutils==0.14
+dogpile.cache==0.7.1
+enum34==1.1.6
+filelock==3.0.10
+funcsigs==1.0.2
+functools32==3.2.3.post2
+futures==3.2.0
+girder==3.0.0a3
+git+https://github.com/girder/girder.git#egg=girder-autojoin&subdirectory=plugins/autojoin
+git+https://github.com/girder/girder.git#egg=girder-gravatar&subdirectory=plugins/gravatar
+git+https://github.com/girder/girder.git#egg=girder-jobs&subdirectory=plugins/jobs
+git+https://github.com/girder/girder.git#egg=girder-oauth&subdirectory=plugins/oauth
+git+https://github.com/girder/girder.git#egg=girder-thumbnails&subdirectory=plugins/thumbnails
```

Comment:
All these, isic-challenge-submission, and covalic (above) should have a sha1 sum associated with them.